src/Illuminate/Database/Schema/Grammars/SQLiteGrammar.php

```diff
@@ -727,7 +728,11 @@ protected function typeTimeTz(Fluent $column)
      */
     protected function typeTimestamp(Fluent $column)
     {
-        return $column->useCurrent ? 'datetime default CURRENT_TIMESTAMP' : 'datetime';
+        if ($column->useCurrent) {
+            $column->default(new Expression('CURRENT_TIMESTAMP'));
+        }
+
+        return 'datetime';
```

Comment:
Please explain need for this change.

Reply:
to be consistent with other Grammars, I changed SQLite too to use the same syntax for handling `useCurrent`.

Reply:
there are 2 different statements to change column type and default value on PostgreSQL and SQL server.